### DISCUSSION TITLE
_hw_endpoint_start_next_buffer behaviour when calling with EP0

### DISCUSSION THREAD

**Discussion by conkerkh** [hathach/tinyusb#1313] [General]
A function below when called with endpoint 0 or 0x80 will clearly access Null ptr, since there is no control ep for ep0. Am I missing something or it's a bug?

https://github.com/hathach/tinyusb/blob/d6fb1859a466508534b336c468ab328ea6704894/src/portable/raspberrypi/rp2040/rp2040_usb.c#L142-L178

**Comment by hathach** [comment]
sorry, late to response, I don't quite understand your point. Can you provide the context also a bit of explanation in which it would cause the null ptr access.
